doc/classes/String.xml

```diff
@@ -684,13 +684,24 @@
 		<method name="to_float" qualifiers="const">
 			<return type="float" />
 			<description>
-				Converts a string containing a decimal number into a [code]float[/code].
+				Converts a string containing a decimal number into a [code]float[/code]. The method will stop on the first non-number character except [code].[/code], and [code]e[/code] which is used for exponential.
+				[codeblock]
+				print("12.3".to_float()) # 12.3
+				print("1.2.3".to_float()) # 1.2.3
```

Comment:
Maybe you meant `1.2` here? Otherwise this is really not expected behavior.
```suggestion
				print("1.2.3".to_float()) # 1.2
```

Maybe there's a good reason to it, but I don't see why this function should return anything than `null` or `nan` or some other response to indicate a parsing error on what is an invalid input.

But for now, properly documenting quirks is nice, even if the function might eventually be changed...

Reply:
> Did you mean 1.2 here?

Yes...